build.gradle

```diff
@@ -8,16 +14,18 @@ allprojects {
 }
 
 apply plugin: 'java'
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
 
 tasks.withType(JavaCompile) {
-	options.encoding = 'UTF-8'
+    options.encoding = 'UTF-8'
+}
+javafx {
+    version = "19"
+    modules = [ 'javafx.controls', 'javafx.fxml', 'javafx.web', 'javafx.swing' ]
```

Comment:
How does this now work in the modularized world? Does this adds these modules twice now, once as dependency of afterburner and once as dep of the main application? Are they combined?

In either case, the web and swing modules shouldn't be needed, or are they?

Reply:
The project afterburner itself is not modularized, I used the shadow jar plugin. It generates two files:
One with all deps inside and one  minimal shadow jar that only contains the classes itself.  I used the latter to include it in jabref. 

Main reason to avoid name clashes with the java ee stuff for the rest api
https://github.com/koppor/jabref/pull/628